lib/wallpapers.nix

```
@@ -27,4 +27,16 @@
       };
     };
   };
+
+  # Values are taken from
+  #  https://invent.kde.org/plasma/kdeplasma-addons/-/blob/bc53d651cf60709396c9229f8c582ec8a9d2ee53/applets/mediaframe/package/contents/ui/ConfigGeneral.qml#L148-170
+  wallpaperFillModeTypes = {
+    "Stretch" = 0; # a.k.a. Scaled
+    "PreserveAspectFit" = 1; # a.k.a. Scaled Keep Proportions
+    "PreserveAspectCrop" = 2; # a.k.a. Scaled And Cropped
+    "Tile" = 3;
+    "TileVertically" = 4;
+    "TileHorizontally" = 5;
+    "Pad" = 6; # a.k.a. Centered
+  };
```

```suggestion
  wallpaperFillModeTypes = {
    "stretch" = 0; # a.k.a. Scaled
    "preserveAspectFit" = 1; # a.k.a. Scaled Keep Proportions
    "preserveAspectCrop" = 2; # a.k.a. Scaled And Cropped
    "tile" = 3;
    "tileVertically" = 4;
    "tileHorizontally" = 5;
    "pad" = 6; # a.k.a. Centered
  };
```

Settings values are should be named in camelCase for consistency